src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -108,13 +108,15 @@ func newBootstrapCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 						Gateway: apiv1.GatewayConfig{
 							Enabled: vals.Pointer(true),
 						},
-						MetricsServer: apiv1.MetricsServerConfig{
-							Enabled: vals.Pointer(true),
-						},
 					},
 				}
 			}
 
+			// Always enable MetricsServer by default
+			if bootstrapConfig.ClusterConfig.MetricsServer.Enabled == nil {
+				bootstrapConfig.ClusterConfig.MetricsServer.Enabled = vals.Pointer(true)
+			}
+
```

Comment:
we can revert this, as it is handled in https://github.com/canonical/k8s-snap/pull/327/files#diff-b5b56d715e342b6e5c59f5158d6cf255bf5f1ff11f6d7ff786f6d98b8842eb1bR98